### ISSUE TITLE
Include a preset selector and a preview in the config window

### ISSUE THREAD

**Issue by bawr** [AlexIIL/BetterLoadingScreen#31]
That way, those "preset one / preset two" files would actually be more useful, and it would allow users to add and test their own presets without having to reload Minecraft.

(Okay, so I did think of something actually useful, heh.)

**Comment by AlexIIL** [comment]
Thats a good idea. Its already implemented in my 1.4.x branch so i guess i just need to backport it.

**Comment by AlexIIL** [comment]
Although you already have to reload minecraft to test the preset- its a loading screen

**Comment by bawr** [comment]
Yeah, that's why an additional preview button would be useful, just displaying the loading screen for a few seconds, or until a key is pressed.

**Comment by AlexIIL** [comment]
ooooh i missed out the reviewer part

**Comment by AlexIIL** [comment]
yea, thats definatly a good idea, Although ideally It would be a GUI designer (add elements via a gui- a bit like power point with images ond objects)

**Comment by bawr** [comment]
Right, but a peview feature is a good first step, and useful in its own right.

**Comment by AlexIIL** [comment]
True

**Comment by AlexIIL** [comment]
Ok, would something like this be ok? (Editor coming afterwards)
![screenshot from 2015-08-01 21 12 37](https://cloud.githubusercontent.com/assets/3751138/9023421/2a889fc4-3892-11e5-86b4-b9b1d959155d.png)

**Comment by AlexIIL** [comment]
And I realised I should include a preset selector. Just adding it.

**Comment by AlexIIL** [comment]
And the text box is huge. I know that too.

**Comment by AlexIIL** [comment]
Wow sorry for not doing this earlier, I got sidetracked by BuildCraft... I’m getting back to BLS now though so expect this within a week (The preview part, I’m not making any guarantees about the editor)
